lib/decorators/party_name_attributes.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'scraped'
+
+# Members on the members page are grouped by party but the names or ids of the
+# parties are not given. Instead, each party is signified by an image.
+
+# This decorator maps the party name and id to the image name and populates the
+# party image <img> tags with party_name and party_id attributes.
+class PartyNameAttributes < Scraped::Response::Decorator
+  PARTY_IMAGES_MAPPED =
+    {
```

Comment:
splitting this declaration over two lines like this is a little odd and mildly confusing.